src/scm/gitlab.js

```diff
@@ -199,31 +200,31 @@ class Gitlab {
         contentType: 'application/json',
         data: JSON.stringify(payload)
       })
-        .then(resolve)
-        .fail(reject);
+      .then(resolve)
+      .fail(reject);
     })
-      .then(response => {
-        const repo = {
-          fullName: response.name,
-          id: response.id
-        };
-        context.repo = repo;
-        Object.assign(context.bindRepo, {[context.id]: repo});
-        if (context.bindBranch[context.id]) {
-          delete context.bindBranch[context.id];
-        }
-        chrome.storage.sync.set({bindRepo: context.bindRepo});
-        return response.name;
-      })
-      .then(repo => {
-        return this.commitFiles(repo, 'master', null, [{name: "README.md", content: "initialed by gas-github"}], null,null, 'initial commit')
-          .then(() => {
-            return repo;
-          });
+    .then(response => {
+      const repo = {
+        fullName: response.path_with_namespace,
+        id: response.id
+      };
+      context.repo = repo;
+      Object.assign(context.bindRepo, {[context.id]: repo});
+      if (context.bindBranch[context.id]) {
+        delete context.bindBranch[context.id];
+      }
+      chrome.storage.sync.set({bindRepo: context.bindRepo});
+      return response.path_with_namespace;
+    })
+    .then(repo => {
+      return this.commitFiles(repo, 'master', null, [{name: "README.md", content: "initialed by gas-github"}], null,null, 'initial commit')
+        .then(() => {
+          return repo;
+        });
       })
-      .catch((err) => {
-        throw new Error('Failed to create new repository.');
-      });
+    .catch((err) => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/c0b889de942411e3a53c4f3be8d2b855a8d0621a/src/scm/gitlab.js#L225"></span>‘err’ is defined but never used.</td>
    </tr>
  </tbody>
</table>